bitnami/wordpress/Chart.yaml

```diff
@@ -35,4 +35,4 @@ name: wordpress
 sources:
   - https://github.com/bitnami/bitnami-docker-wordpress
   - http://www.wordpress.com/
```

Comment:
Could you rebase of master? the URL was changed http://www.wordpress.com/ to http://www.wordpress.org/

Reply:
Done